Doc/library/datetime.rst

```diff
@@ -985,9 +985,10 @@ Other constructors, all class methods:
    is used.
 
    For any :class:`.datetime` object *d*,
-   ``d == datetime.combine(d.date(), d.time(), d.tzinfo)``. If date is a
-   :class:`.datetime` object, its time components and :attr:`.tzinfo` attributes
-   are ignored.
+   ``d == datetime.combine(d.date(), d.time(), d.tzinfo)``.
+
+   If the *date* argument is a :class:`.datetime` object, its time components and
+   :attr:`.tzinfo` attributes are ignored.
```

Comment:
In my opinion, the item about datetime identity should come last, since it's merely nice to know.

If someone concurs, I'll make the change.

Reply:
@belm0 - thank you for your contribution.  This PR is a nice improvement to the text of documentation and I would like to merge it without much further changes.  Yes, placing the  "For any datetime object d..." part in the concluding paragraph makes sense.  I would consider keeping the doc in two paragraphs:

... If the *tzinfo* argument is provided, ... If the *date* argument is ...

For any [datetime](https://docs.python.org/3.12/library/datetime.html#datetime.datetime) object ...

Reply:
done, thank you